src/common/SurgePatch.cpp

```diff
@@ -810,7 +810,7 @@ void SurgePatch::do_morph()
 struct patch_header
 {
    char tag[4];
-   unsigned int xmlsize, wtsize[2][3];
+   unsigned int xmlsize, wtsize[n_scenes][n_oscs];
```

Comment:
This one is super tricky since this is used in streaming and read directly as a binary. So this change means that if we increase n_osc no old patches will load. Can you back this one down and add a comment?

Reply:
Can do!